components/encryption/src/errors.rs

```diff
@@ -18,6 +18,18 @@ pub enum Error {
     Crypter(CrypterError),
     #[fail(display = "Protobuf error {}", _0)]
     Proto(ProtobufError),
+    #[fail(display = "Unknown encryption error")]
+    UnknownEncryption,
+    #[fail(
+        display = "Current master key error {}, previous master key error {}.",
+        _0, _1
+    )]
+    DictDecrypt(
+        Box<dyn error::Error + Sync + Send>,
+        Box<dyn error::Error + Sync + Send>,
+    ),
+    #[fail(display = "Encrypted file error {}", _0)]
+    EncryptedFile(Box<dyn error::Error + Sync + Send>),
```

Comment:
Are they still necessary as we know whether the master is correct or not by checking sha256?